src/bench_tools.py

```diff
@@ -22,6 +23,12 @@ def rados_bench(
         return _output.decode("UTF-8")
 
     def rbd_remove_image(self, pool_name):
+        # first unmap the image otherwise removing the image will fail
+        _rbd_name = f"/dev/rbd/{ pool_name }/{ self.charm_instance.RBD_IMAGE }"
+        if os.path.exists("/dev/rbd0"):
```

Comment:
why not check that `_rbd_name` exists? It's the one that is going to be unmap'd after all.

Reply:
That's a fair point. I used the variable name instead and tested successfully on both Focal and Jammy